src/c++/perf_analyzer/genai-perf/genai_perf/utils.py

```diff
@@ -49,7 +53,22 @@ def load_yaml(filepath: Path) -> Dict[str, Any]:
 
 def load_json(filepath: Path) -> Dict[str, Any]:
     with open(str(filepath), encoding="utf-8", errors="ignore") as f:
-        return json.load(f)
+        content = f.read()
+        try:
+            return json.loads(content)
+        except json.JSONDecodeError:
+            snippet = content[:200] + ("..." if len(content) > 200 else "")
+            logger.error("Failed to parse JSON string: '%s'", snippet)
+            raise
+
+
+def load_json_str(json_str: str) -> Dict[str, Any]:
```

Comment:
Is `load_json` ever used in the code? Or am I not seeing them because PR only shows the ones that changed to use `load_json_str`?

Reply:
It is. The latter. It's used here: https://github.com/triton-inference-server/client/blob/ade066d486498c5e5dd0cb701cca81bcaddb24c6/src/c%2B%2B/perf_analyzer/genai-perf/genai_perf/profile_data_parser/profile_data_parser.py#L52